app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] @sagieramos and @Samuelwanza Great work adding a form to login  users, however, the styling of this page doesn't match the wireframe design given, check the comparison below:

Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/sagieramos/blogpost/assets/61408860/624ac548-7d3f-40cf-a238-16eab3170ddd)  | ![image](https://github.com/sagieramos/blogpost/assets/61408860/34bb736e-c85c-411c-b178-18f58c11b0e0)

Reply:
Thanks for pointing this out. I have made the necessary changes by replacing the placeholder of each field with its supposed label. 
In other to make the UI look the same as the wireframe, I have removed the signup button used in adding a new user. The sign_up button will be shown in the UI when a sign_in fails due to credentials not being present in the database